pkg/function/client/client_resolver.go

```diff
@@ -0,0 +1,67 @@
+package client
+
+import (
+	"github.com/numaproj/numaflow-go/pkg/function"
+	"google.golang.org/grpc/resolver"
+	"log"
+	"strconv"
+	"strings"
+)
+
+const (
+	custScheme      = "example"
+	custServiceName = "lb.example.grpc.io"
+	connAddr        = "0.0.0.0"
+)
+
+var addrsList []string
```

Comment:
As we discussed offline, this can be moved into `multiProcResolverBuilder` as a property.

Reply:
Updated